scripts/manageacl.py

```diff
@@ -0,0 +1,158 @@
+import argparse
+import asyncio
+from enum import Enum
+
+from azure.core.credentials import AzureKeyCredential
+from azure.core.credentials_async import AsyncTokenCredential
+from azure.identity import AzureDeveloperCliCredential
+from azure.search.documents.aio import SearchClient
+from azure.search.documents.indexes.aio import SearchIndexClient
+from azure.search.documents.indexes.models import SearchFieldDataType, SimpleField
+
+class ManageAcl:
+    async def __aenter__(self):
+        endpoint = f"https://{self.service_name}.search.windows.net"
+        self.search_client = SearchClient(endpoint=endpoint, credential=self.credentials, index_name=self.index_name)
+        await self.search_client.__aenter__()
+        self.search_index_client = SearchIndexClient(endpoint=endpoint, credential=self.credentials)
+        await self.search_index_client.__aexit__()
+        return self
+
+    async def __aexit__(self, *args, **kwargs):
+        await self.search_client.__aexit__(*args, **kwargs)
+        await self.search_index_client.__aexit__(*args, **kwargs)
```

Comment:
why does it exit search_index_client twice? (here and above?)

Reply:
this was a mistake - but i'll remove this per your other feedback